.github/workflows/cd.yml

```diff
@@ -453,7 +453,7 @@ jobs:
             }
 
             // Parse and filter environments
-            const allowedEnvironments = ['dev', 'ops', 'prod'];
+            const allowedEnvironments = ['dev', 'ops', 'prod-canary', 'prod'];
```

Comment:
@xnyo We are adding prod-canary only to be included when prod is passed, right?

Reply:
When we pass target, it publishes the plugin to that env's catalog and we don't have prod-canary catalog(it is prod), not exactly sure how that works?

Reply:
When target is prod-canary:
1. Publish to prod catalog - Would this cause any errors?
2. Trigger argo workflow - this will work fine
3. Should we publish the docs?
After deploying to prod-canary, it is called with prod
1. Publishing again to prod - Would this cause errors? Duplicate version error?